Library/Homebrew/formula_auditor.rb

```diff
@@ -802,7 +802,7 @@ def audit_specs
         tag = SharedAudits.github_tag_from_url(url)
         tag ||= formula.stable.specs[:tag]
 
-        if @online
+        if @online && !tag.nil?
```

Comment:
Technically this only applies to `luajit`, since we expect everything else to build from a tag. It seems like there should be a nicer/better way to handle this given that.

But I don't mind merging this for now to unblock the Homebrew/core PR.

Reply:
the SharedAudit checks are tag checks anyway